packages/p2p-media-loader-core/src/segments-storage.ts

```diff
@@ -101,7 +105,13 @@ export class SegmentsMemoryStorage {
   }
 
   // eslint-disable-next-line @typescript-eslint/require-await
-  private async clear(): Promise<boolean> {
+  private async clear(isLiveStream: boolean): Promise<boolean> {
+    const cacheSegmentExpiration =
+      (this.storageConfig.cachedSegmentExpiration ??
+        (isLiveStream ? 60 * 20 : 0)) * 1000;
```

Comment:
Please make a module level constant
```suggestion
        (isLiveStream ? DEFAULT_LIVE_CACHED_SEGMENT_EXPIRATION : 0)) * 1000;
```